### PR TITLE
[stable/postgresql ] Load Balancer sources

### PR DESCRIPTION
**Pull request by jmvizcainoio**
Signed-off-by: Juan Manuel <juanmanuel.vizcaino@iomed.es>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/postgresql/templates/svc.yaml [hunk 1]
```diff
@@ -16,6 +16,10 @@ spec:
   {{- if and .Values.service.loadBalancerIP (eq .Values.service.type "LoadBalancer") }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
   {{- end }}
+  {{- if .Values.service.loadBalancerSourceRanges }}
```
Comment (juan131) [inline stable/postgresql/templates/svc.yaml]: Shouldn't you check if the service.type is a **LoadBalancer** too?

```yaml
{{- and (eq .Values.service.type "LoadBalancer") .Values.service.loadBalancerSourceRanges }}
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for your contribution @jmvizcainoio !
Please take a look to my comments when you have a chance

File: stable/postgresql/templates/svc.yaml (hunk 1 above)
Comment (jmvizcainoio) [reply stable/postgresql/templates/svc.yaml]: Yes, better 👍 I changed to check.

File: stable/postgresql/values-production.yaml [hunk 2]
```diff
@@ -180,6 +180,11 @@ service:
   ##
   # loadBalancerIP:
 
+  ## Load Balancer sources
+  ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  # loadBalancerSourceRanges:
```
Comment (carrodher) [inline stable/postgresql/values-production.yaml:186]: Can you add a line only with ## between the comment and the parameter? Check other examples:
```yaml
  ## RabbitMQ Manager port
  ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
  ##
  managerPort: 15672
```

File: stable/postgresql/values.yaml [hunk 3]
```diff
@@ -187,6 +187,11 @@ service:
   ##
   # loadBalancerIP:
 
+  ## Load Balancer sources
+  ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  # loadBalancerSourceRanges:
+  #  - 10.10.10.0/24
```
Comment (carrodher) [inline stable/postgresql/values.yaml]: Can you add a line only with ## between the comment and the parameter? Check other examples:
```yaml
  ## RabbitMQ Manager port
  ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
  ##
  managerPort: 15672
```

Comment (jmvizcainoio) [reply stable/postgresql/values.yaml]: ok added

File: stable/postgresql/values-production.yaml (hunk 2 above)
Comment (jmvizcainoio) [reply stable/postgresql/values-production.yaml:186]: ok added

**Pull request merged**